scripts/e2e-tests.sh

```diff
@@ -5,10 +5,12 @@ MYDIR=$(realpath "$(dirname "$0")")
 cd "$MYDIR"
 cd ..
 
-NIGHTLY_TOOLCHAIN=${NIGHTLY_TOOLCHAIN:-nightly}
-cargo +"$NIGHTLY_TOOLCHAIN" build --release --target wasm32-unknown-unknown -Z build-std=std,panic_abort -Z build-std-features=panic_immediate_abort
+# We should use stable here once nitro-testnode is updated and the contracts fit
+# the size limit.
+RUSTUP_TOOLCHAIN=${RUSTUP_TOOLCHAIN:-nightly}
+
+cargo build --release --target wasm32-unknown-unknown -Z build-std=std,panic_abort -Z build-std-features=panic_immediate_abort
```

Comment:
Those are nightly compiler optimization flags. As I remember these flags been recommended for wasm size optimization.
I haven't tried to experiment with nightly flags further, since wasm size is fine for us.